pallets/identity-management/src/tests.rs

```diff
@@ -107,17 +107,17 @@ fn link_identity_with_unauthorized_delegatee_fails() {
 }
 
```

Comment:
would it be better to add another test for `activate_identity_works()` ?

Reply:
Yes, I'll add it, thanks